### PR TITLE
feat: Developer Guide - GUI - MySQL Workbench

### PR DESCRIPTION
**Pull request by shawn0915**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add develop/dev-guide-gui-mysql-workbench.md

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/15366, https://github.com/pingcap/docs/pull/16126, https://github.com/pingcap/docs/pull/16127
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Icemap — APPROVED** [review]
LGTM

File: develop/dev-guide-gui-mysql-workbench.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+    ![MySQL Workbench: add new connection](/media/develop/mysql-workbench-add-new-connection.png)
+
+6. 在 **Setup New Connection** 对话框中，配置以下连接参数：
+
+    - **Connection Name**：为该连接指定一个有意义的名称。
+    - **Hostname**：输入从 TiDB Cloud 连接对话框中的得到的 `HOST` 参数。
+    - **Port**：输入从 TiDB Cloud 连接对话框中的得到的 `PORT` 参数。
+    - **Username**：输入从 TiDB Cloud 连接对话框中的得到的 `USERNAME` 参数。
+    - **Password**：点击 **Store in Keychain ...**，输入 TiDB Serverless 集群的密码，然后点击 **OK** 保存密码。
```
Comment (qiancai) [inline develop/dev-guide-gui-mysql-workbench.md:67]:
```suggestion
    - **Password**：点击 **Store in Keychain ...**  或 **Store in Vault**，输入 TiDB Serverless 集群的密码，然后点击 **OK** 保存密码。
```

Comment (qiancai) [reply develop/dev-guide-gui-mysql-workbench.md:67]: We can cover https://github.com/pingcap/docs/pull/16136 in this translation PR too😂

File: develop/dev-guide-gui-mysql-workbench.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    ![MySQL Workbench: configure connection settings for TiDB Serverless](/media/develop/mysql-workbench-connection-config-serverless-parameters.png)
+
+7. 点击 **Test Connection** 以验证与 TiDB Serverless 集群的连接。
+
+8. 如果连接测试成功，你可以看到 **Successfully made the MySQL connection** 信息。点击 **OK** 保存连接配置。
+
+</div>
+<div label="TiDB Dedicated">
+
+1. 在 TiDB Cloud 的 [**Clusters**](https://tidbcloud.com/console/clusters) 页面中，点击你目标集群的名字，并进入集群的 Overview 页面。
```
Comment (qiancai) [inline develop/dev-guide-gui-mysql-workbench.md:82]:
```suggestion
1. 在 TiDB Cloud 的 [**Clusters**](https://tidbcloud.com/console/clusters) 页面中，点击你目标集群的名字，进入集群的 **Overview** 页面。
```

File: develop/dev-guide-gui-mysql-workbench.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+2. 点击右上角的 **Connect** 按钮，将会弹出连接对话框。
+
+3. 点击 **Allow Access from Anywhere**。
+
+    有关如何获取连接字符串的更多详细信息，参考 [TiDB Dedicated 标准连接](https://docs.pingcap.com/tidbcloud/connect-via-standard-connection)。
```
Comment (qiancai) [inline develop/dev-guide-gui-mysql-workbench.md:88]:
```suggestion
    有关如何获取连接字符串的更多详细信息，参考 [TiDB Dedicated 标准连接（英文）](https://docs.pingcap.com/tidbcloud/connect-via-standard-connection)。
```

File: develop/dev-guide-gui-mysql-workbench.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+4. 启动 MySQL Workbench，并点击 **MySQL Connections** 标题附近的 **+**。
```
Comment (qiancai) [inline develop/dev-guide-gui-mysql-workbench.md:90]:
```suggestion
4. 启动 MySQL Workbench，并点击 **MySQL Connections** 标题旁边的 **+**。
```

File: develop/dev-guide-gui-mysql-workbench.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+    ![MySQL Workbench: configure connection settings for TiDB Dedicated](/media/develop/mysql-workbench-connection-config-dedicated-parameters.png)
+
+6. 点击 **Test Connection** 以验证与 TiDB Dedicated 集群的连接。
+
+7. 如果连接测试成功，你可以看到 **Successfully made the MySQL connection** 信息。点击 **OK** 保存连接配置。
+
+</div>
+<div label="TiDB Self-Hosted">
+
+1. 启动 MySQL Workbench，并点击 **MySQL Connections** 标题附近的 **+**。
```
Comment (qiancai) [inline develop/dev-guide-gui-mysql-workbench.md:115]:
```suggestion
1. 启动 MySQL Workbench，并点击 **MySQL Connections** 标题旁边的 **+**。
```

File: develop/dev-guide-gui-mysql-workbench.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+4. 如果连接测试成功，你可以看到 **Successfully made the MySQL connection** 信息。点击 **OK** 保存连接配置。
+
+</div>
+</SimpleTab>
+
+## 下一步
+
+- 关于 MySQL Workbench 的更多使用方法，可以参考 [MySQL Workbench 官方文档](https://dev.mysql.com/doc/workbench/en/)。
+- 你可以继续阅读 [开发者文档](/develop/dev-guide-overview.md)，以获取更多关于 TiDB 应用开发的最佳实践。例如：[插入数据](/develop/dev-guide-insert-data.md)、[更新数据](/develop/dev-guide-update-data.md)、[删除数据](/develop/dev-guide-delete-data.md)、[单表读取](/develop/dev-guide-get-data-from-single-table.md)、[事务](/develop/dev-guide-transaction-overview.md)、[SQL 性能优化](/develop/dev-guide-optimize-sql-overview.md)等。
```
Comment (qiancai) [inline develop/dev-guide-gui-mysql-workbench.md:143]:
```suggestion
- 你可以继续阅读[开发者文档](/develop/dev-guide-overview.md)，以获取更多关于 TiDB 应用开发的最佳实践。例如：[插入数据](/develop/dev-guide-insert-data.md)、[更新数据](/develop/dev-guide-update-data.md)、[删除数据](/develop/dev-guide-delete-data.md)、[单表读取](/develop/dev-guide-get-data-from-single-table.md)、[事务](/develop/dev-guide-transaction-overview.md)、[SQL 性能优化](/develop/dev-guide-optimize-sql-overview.md)等。
```

File: develop/dev-guide-gui-mysql-workbench.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+- 如果你更倾向于参与课程进行学习，我们也提供专业的 [TiDB 开发者课程](https://cn.pingcap.com/courses-catalog/category/back-end-developer/?utm_source=docs-cn-dev-guide) 支持，并在考试后提供相应的 [资格认证](https://learn.pingcap.com/learner/certification-center)。
```
Comment (qiancai) [inline develop/dev-guide-gui-mysql-workbench.md:144]:
```suggestion
- 如果你更倾向于参与课程进行学习，我们也提供专业的 [TiDB 开发者课程](https://cn.pingcap.com/courses-catalog/category/back-end-developer/?utm_source=docs-cn-dev-guide) 支持，并在考试后提供相应的[资格认证](https://learn.pingcap.com/learner/certification-center)。
```

File: develop/dev-guide-gui-mysql-workbench.md
Comment (qiancai) [inline develop/dev-guide-gui-mysql-workbench.md:1]: preview: https://pingcap-docsite-preview-git-pr-qiancai.vercel.app/zh/tidb/dev/dev-guide-gui-mysql-workbench

**Pull request merged**